src/endpoints/load_balancing/delete_pool.rs

```diff
@@ -0,0 +1,29 @@
+use crate::framework::endpoint::{Endpoint, Method};
+use crate::framework::response::ApiResult;
+
+/// Delete Pool
+/// https://api.cloudflare.com/#account-load-balancer-pools-delete-pool
+pub struct DeletePool<'a> {
+    /// The Cloudflare account to create this Pool under.
+    pub account_identifier: &'a str,
+    /// Which pool to retrieve the details of.
+    pub identifier: &'a str,
```

Comment:
The parameter comments should be updated to reflect the semantics of DeletePool –– e.g. "Which pool to delete"

Reply:
Thanks, fixed